TestCases/turbomachinery/Aachen_turbine/aachen_3D_MP.cfg

```diff
@@ -0,0 +1,363 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: AACHEN turbine 3D                                          %
+% Author: S. Vitale, A. Cappiello                                              %
+% Institution: Delft University of Technology                                  %
+% Date: Oct 20th, 2023                                                         %
+% File Version 8.0 develop                                        %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+%
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES,
+%                               WAVE_EQUATION, HEAT_EQUATION, LINEAR_ELASTICITY,
+%                               POISSON_EQUATION)                         
```

Comment:
Please remove this type of option list, it becomes stale and it's hard to maintain, document only what is important for this specific case.

Reply:
I've removed the most cumbersome comments with long and outdate options